lib/model/testos_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright (C) 2019 The Syncthing Authors.
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package model
+
+import (
+	"os"
+	"time"
+)
+
+// fatal is the required common interface between *testing.B and *testing.T
+type fatal interface {
+	Fatal(...interface{})
+}
+
+type fatalOs struct {
```

Comment:
call it testOs that takes testing.T?

Reply:
It should also work with benchmarks, that's why I added the `fatal` interface. And I don't like naming the variable (`testOS`) the same way as the type (`fatalOs`). Does that make sense or would you use other names (or the same)?